web/src/components/Participant.js

```diff
@@ -0,0 +1,212 @@
+import React, { Component } from 'react'
+// import { Button, Intent, Spinner } from "@blueprintjs/core";
+
+class Form extends Component {
+    constructor(props) {
+        super(props)
+        this.state = {
+            email: '',
+            names: '',
+            city: '',
+            choosing: 'Salesforce',
+            skill: '',
+            position: '',
+            company: '',
+            help: '',
+            feedBack: '',
+            authorization: '',
+            roles: ''
+        }
+    }
+    handleEmailChange = event => {
+        this.setState({
+            email: event.target.value
+        })
+    }
+    handleNames = event => {
+        this.setState({
+            names: event.target.value
+        })
+    }
+    handleCityChange = event => {
+        this.setState({
+            city: event.target.value
+        })
+    }
+    handleNames = event => {
+        this.setState({
+            names: event.target.value
+        })
+    }
+    handleResume = event => {
+        this.setState({
+            city: event.target.value
+        })
+    }
+    handleJob= event => {
+        this.setState({
+            choosing: event.target.value
+        })
+    }
+    handlePosition = event => {
+        this.setState({
+            position: event.target.value
+        })     
+    }
+    handleSkills = event => {
+        this.setState({
+            skill: event.target.value
+        })
+    }
+    handleCompany = event => {
+        this.setState({
+            company: event.target.value
+        })
+    }
+
+    handleHelp = event => {
+        this.setState({
+            help: event.target.value
+        })
+    }
+
+    handleFeedback = event => {
+        this.setState({
+            feedBack: event.target.value
+        })
+    }
+
+    handleAuthorization = event => {
+        this.setState({
+            authorization: event.target.value
+        })
+    }
+    
+    handleRoles = event => {
+        this.setState({
+            roles: event.target.value
+        })
+    }
+
+    handleSubmit = event => {
+        alert(`${this.state.name} ${this.state.comments} ${this.state.choosing}`)
+        event.preventDefault()
+    }
```

Comment:
If you change your state to be an object, all of this could become one handler that takes a key of the <input> and looks that up in the state file and sets it accordingly.